src/react/workflow/workflow/Workflow.jsx

```diff
@@ -0,0 +1,132 @@
+//@flow
```

Comment:
Let's move `src/react/workflow/workflow/Workflow.jsx` to `src/react/workflow/WorkflowEditor.jsx`